.projenrc.js

```diff
@@ -53,6 +53,13 @@ const project = new awscdk.AwsCdkConstructLibrary({
   },
 });
 
+// We only need aws-cdk-lib and constructs for testing. Neither library is used
+// in the public API.
+project.deps.removeDependency('constructs', DependencyType.PEER);
+project.deps.addDependency('constructs@10.0.5', DependencyType.DEVENV);
```

Comment:
I used explicit versions here to prevent projen from auto-upgrading to the latest. We can make that update in a separate PR.